Update interfaces/cookie-store.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/a9fccc7/whatwg/idl/cookie-store.idl

foolip commented:
This PR is removing cookie-store.idl because it isn't in reffy-reports, which should be fixed by getting https://wicg.github.io/cookie-store/ scraped.

foolip commented:
I've sent https://github.com/tidoust/reffy/pull/124. If it's merged, this PR should either be updated to reflect changes or be closed automatically.

foolip committed:
Update interfaces/cookie-store.idl

Source: https://github.com/tidoust/reffy-reports/blob/a9fccc7/whatwg/idl/cookie-store.idl

foolip reviewed (approved):
I've confirmed that the tests still work. There are 2 tests fewer, but it's not broken.
